aries-site/src/layouts/main/Layout.js

```diff
@@ -10,7 +10,9 @@ const calcPad = size => {
   return val;
 };
 
-export const Layout = ({ children, title, isLanding }) => {
+export const Layout = ({ children, descriptiveHeader, title, isLanding }) => {
+  // If a DescriptiveHeader is provided, then it should be a NavPage
+  const isNavPage = descriptiveHeader;
```

Comment:
I wonder if would be clearer to explicitly state `isNavPage` as a boolean prop, rather that inferring type of page layout based on an element being passed or not?

Reply:
I am open to that as well. I had that initially actually, but had changed it but maybe for clarity it would be best to define it as a prop.